diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index d59ba34..4389f74 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -728,12 +728,7 @@ if (__DEV__) {
   Object.defineProperty(module.exports, 'DatePickerAndroid', {
     configurable: true,
     get() {
-      invariant(
-        false,
-        'DatePickerAndroid has been removed from React Native. ' +
-          "It can now be installed and imported from '@react-native-community/datetimepicker' instead of 'react-native'. " +
-          'See https://github.com/react-native-datetimepicker/datetimepicker',
-      );
+      return require('@react-native-picker/picker')
     },
   });
 }
